### PR TITLE
- Deprecated support for Python 3.3. See #2477

### LINKED ISSUE #2477: Deprecate Python 3.3 support in Pyramid 1.7
Deprecate Python 3.3 support in Pyramid 1.7. Add tasks below by editing this comment. See #2368 and #2476.

- [ ] Add note to CHANGES.txt.

### PR DESCRIPTION
**Pull request by stevepiercy**
Closes #2477

### CODE REVIEW

**Comment by stevepiercy** [comment]
ping @mmerickel @bertjwregeer anyone else?

File: CHANGES.txt [hunk 1]
```diff
@@ -1,6 +1,9 @@
 unreleased
 ==========
 
+- Deprecated support for Python 3.3.
```
Comment (tseaver) [inline CHANGES.txt]: The changelog entry should state when support will be removed, e.g.:

```
- (Deprecation) Support for Python 3.3 will be removed in Pyramid 1.8.
```

**Comment by stevepiercy** [comment]
Thanks @tseaver

**Comment by stevepiercy** [comment]
Is it OK to merge this, or are there other tasks that need to be done?

**Comment by mmerickel** [comment]
@tseaver any recommendations on deprecating a version other than adding a note to the changelog? Is it worth adding something at runtime or in setup.py (which isn't run when installed as a wheel)?

**Comment by tseaver** [comment]
@mmerickel I wouldn't add anything beyond the changelog entry, myself.

**Comment by bertjwregeer** [comment]
We should drop this into whatsnew-1.7 as well... outside of that I think we are good :-)

**Comment by mmerickel** [comment]
Sounds good, I just wanted to get a second opinion!

**Pull request merged**